annotation-processor/src/main/java/com/bloxbean/cardano/client/plutus/annotation/blueprint_processor/JavaFileUtil.java

```diff
@@ -0,0 +1,59 @@
+package com.bloxbean.cardano.client.plutus.annotation.blueprint_processor;
+
+import com.bloxbean.cardano.client.plutus.blueprint.model.BlueprintDatatype;
+import com.bloxbean.cardano.client.plutus.blueprint.model.BlueprintSchema;
+import com.squareup.javapoet.JavaFile;
+import com.squareup.javapoet.TypeSpec;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.annotation.processing.ProcessingEnvironment;
+import javax.tools.JavaFileObject;
+import java.io.IOException;
+import java.io.Writer;
+
+@Slf4j
+public class JavaFileUtil {
```

Comment:
We can move this class to a top-level package util (com.bloxbean.cardano.client.plutus.annotation.processor.util) as this class could be useful for other processors.